package.json

```diff
@@ -84,10 +77,10 @@
     "gh-got": "^2.2.0",
     "istanbul": "^0.4.0",
     "jsdoc": "^3.3.0-beta1",
-    "jsonlint": "^1.6.2",
     "leche": "^2.1.1",
     "linefix": "^0.1.1",
     "load-perf": "^0.2.0",
+    "lodash": "^4.0.0",
```

Comment:
The reason it is duplicate is because both development and production environment is using lodash. Thats not a mistake.